sugar/src/index.ts

```diff
@@ -1,21 +1,20 @@
 import dotenv from 'dotenv'
+import {initSentry} from "./connections/sentry.js";
+import {initChargebee} from "./connections/chargebee.js";
+import {initDiscordWebhookClient} from "./connections/discord.js";
+import {initKafka} from "./connections/kafka.js";
+import {initExpressServer} from "./connections/express.js";
 
-import {Chargebee} from "./connections/chargebee.js";
-import {Koffaka} from "./connections/kafka.js";
-import {Expresso} from "./connections/express.js";
-import {Sentryboo} from "./connections/sentry.js";
-import {DiscordWebhook} from "./connections/discord.js";
 
-dotenv.config()
-
-export const TAG = "Sugar";
 (async () => {
     try {
-        Sentryboo.init()
-        Chargebee.init()
-        DiscordWebhook.init()
-        await Koffaka.init()
-        Expresso.init()
+        dotenv.config()
+
+        initSentry()
+        initChargebee()
+        initDiscordWebhookClient()
+        await initKafka()
+        initExpressServer()
```

Comment:
```suggestion
        dotenv.config();

        initSentry();
        initChargebee();
        initDiscordWebhookClient();
        await initKafka();
        initExpressServer();
```
i'm personally team semi-colon, but maybe i am in the minority XD